native/libcst/src/tokenizer/core/mod.rs

```diff
@@ -940,7 +940,25 @@ impl<'t> TokState<'t> {
                             // skip escaped char (e.g. \', \", or newline/line continuation)
                             self.text_pos.next();
                         }
-                    } else {
+                    } else if let Some(
```

Comment:
Nice, this syntax was finally added in Rust 1.53.0: https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#or-patterns

We could probably clean up the rest of the tokenizer to use this more compact match format.